config/jobs/kubernetes/repo-infra/OWNERS

```diff
@@ -1,9 +1,6 @@
 reviewers:
-- BenTheElder
-- clarketm
-- fejta
-- mikedanese
-approvers:
+- capanato
```

Comment:
everyone in this list is inactive here for the past year+, @cpanato is not and this list was empty, so I'm voluntelling into reviewers on track for approver 🙃